harmonica/forward/point_mass.py

```diff
@@ -178,9 +178,19 @@ def point_mass_gravity(
     cast = np.broadcast(*coordinates[:3])
     result = np.zeros(cast.size, dtype=dtype)
     # Prepare arrays to be passed to the jitted functions
-    coordinates = (np.atleast_1d(i).ravel().astype(dtype) for i in coordinates[:3])
-    points = (np.atleast_1d(i).ravel().astype(dtype) for i in points[:3])
+    coordinates = np.vstack(
+        tuple(np.atleast_1d(i).ravel().astype(dtype) for i in coordinates[:3])
+    )
+    points = np.vstack(
+        tuple(np.atleast_1d(i).ravel().astype(dtype) for i in points[:3])
+    )
```

Comment:
@santisoler I'm not sure why this is needed. Did you come across an error? Numba should be fine with tuples since we don't ever operate on the entire `coordinates`, only one array at a time. It's worth checking is `np.vstack` isn't re-allocating the memory for the arrays. If it is, then this is definitely something we should avoid.

Reply:
I don't know what happened when I wrote this. Stacking the coordinates and points is completely unnecessary, so I went back to defining them as tuples.